app/src/main/java/com/framgia/music_49/data/repository/SongRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.framgia.music_49.data.repository;
+
+import com.framgia.music_49.data.source.SongDataSource;
+import com.framgia.music_49.data.source.remote.SongRemoteDataCallBack;
+
+public class SongRepository {
+    private static SongRepository mInStance;
```

Comment:
```suggestion
    private static SongRepository sInStance;
```
check coding convention